drivers/python/rethinkdb/_import.py

```diff
@@ -16,6 +16,7 @@
 #json parameters
 json_read_chunk_size = 32 * 1024
 json_max_buffer_size = 128 * 1024 * 1024
+max_nesting_depth = 20
```

Comment:
Since https://github.com/rethinkdb/rethinkdb/issues/2645 is already settled, we should raise the default to 100 here as well.